src/documenters/HtmlDocumenter.ts

```diff
@@ -212,7 +211,7 @@ export class HtmlDocumenter {
       output.push(tag('div', 'signature-heading', 'Signature'));
     }
     signatures.forEach(apiItem => {
-      output.push(tag('pre', 'signature', apiItem.getExcerptWithModifiers()));
+      output.push(tag('pre', 'signature', apiItem.getExcerptWithModifiers().replace(/>/g, '&gt').replace(/</g, '&lt')));

```

Comment:
This only escapes <> characters, I think there are more the needs to be escaped.
I think we can use this library: https://www.npmjs.com/package/he

This is something that I would expect the `html-creator` library to do. We pass in text, and the html library makes sure it is escaped accordingly. Let's see if there are any other options here ...